ibis/mapd/ddl.py

```diff
@@ -0,0 +1,422 @@
+from ibis.sql.compiler import DDL, DML
```

Comment:
Do you have tests for the classes in this file? If not, please add them in a follow-up PR.